src/src/WebServer/WebServer.cpp

```diff
@@ -304,6 +304,9 @@ void WebServerInit()
   web_server.on(F("/upload_json"),       HTTP_POST, handle_upload_json, handleFileUpload);
   web_server.on(F("/wifiscanner_json"),  handle_wifiscanner_json);
 #endif // WEBSERVER_NEW_UI
+#ifdef SHOW_SYSINFO_JSON
+    web_server.on(F("/sysinfo_json"),      handle_sysinfo_json);
```

Comment:
It is now added twice, so remove it from the `WEBSERVER_NEW_UI` as the `SHOW_SYSINFO_JSON` is by default defined when the new UI define is defined.

See:

https://github.com/letscontrolit/ESPEasy/blob/ae0e723fe383b227e1013833d66cf9e7915e43ad/src/src/WebServer/WebServer.cpp#L295-L309

Reply:
thanks for catching that.